bpython/curtsiesfrontend/coderunner.py

```diff
@@ -95,60 +93,63 @@ def __init__(self, interp=None, request_refresh=lambda: None):
         """
         self.interp = interp or code.InteractiveInterpreter()
         self.source = None
-        self.main_context = greenlet.getcurrent()
-        self.code_context = None
+        self.code_thread = None
```

Comment:
The intention of 305773234fde003a00e115d6858de82dd93b76e9 and 2b8c30b94690941c00ae8d70a6fe11d5d43d8461 was to use generic names so we do not have to rename them every time we change the implementation.